tests/src/system/basic/WskBasicTests.scala

```diff
@@ -339,24 +339,6 @@ class WskBasicTests
             }
     }
 
-    /**
-     * Tests creating an nodejs action that throws a whisk.error() response. The error message thrown by the
-     * whisk.error() should be returned.
-     */
-    it should "create and invoke a blocking action resulting in a whisk.error response" in withAssetCleaner(wskprops) {
```

Comment:
there are actually three versions now for testing that returning {error: ....} works as expected.  I had left the test that uses the applicationError3.js file and had removed the test that used the other two since they are now essentially all testing the same thing.  We only need one of these now.  I will update the test I had left to say applicationError1.js and remove the other files that are no longer necessary.